src/Cron_Event_Command.php

```diff
@@ -225,22 +225,26 @@ public function run( $args, $assoc_args ) {
 			WP_CLI::error( $events );
 		}
 
+		$hooks = wp_list_pluck( $events, 'hook' );
+		foreach( $args as $hook ) {
+			if ( ! in_array( $hook, $hooks ) ) {
```

Comment:
It is recommended to always use strict mode with `in_array()` (third argument set to `true`).